pythran/tables.py

```diff
@@ -873,6 +873,12 @@ def update_effects(self, node):
     logger.warn("Pythran support disabled for module: omp")
     del MODULES["omp"]
 
+# copyto and isclose are not always supported by numpy
+unsupported_numpy_methods = ('copyto', 'isclose')
```

Comment:
Why it is wrong to support "more than what is provided by numpy"? I mean, users should be happy to be able to use more functionnalities.